api-client/src/routes/RouteInformationTypes.ts

```diff
@@ -46,7 +46,7 @@ export type NewsRouteInformationType = ParamsType & {
 }
 export type SimpleCityContentFeatureType = ParamsType & {
   // Routes without customizable ids, e.g. '/augsburg/de/disclaimer/
-  route: DisclaimerRouteType | OffersRouteType | SprungbrettOfferRouteType | SearchRouteType
+  route: DisclaimerRouteType | OffersRouteType | SprungbrettOfferRouteType | SearchRouteType | LicensesRouteType
 }
```

Comment:
This is not correct, licenses route is not a city content route. Please adjust.

Reply:
done